e2e/components/Dialog/ActionDialog.test.ts

```diff
@@ -10,7 +10,7 @@ test('ダイアログが開閉できること', async (t) => {
   const trigger = Selector('[data-test=dialog-trigger]')
   const content = Selector('[data-test=dialog-content]')
   const closer = content.find('.smarthr-ui-Dialog-closeButton')
-  const background = content.parent('[role=dialog]').find('.smarthr-ui-Dialog-background')
+  const background = Selector('.smarthr-ui-Dialog-background')
```

Comment:
一応ですがクラス名だけで要素を探すと、複数個ダイアログが存在するときに想定する `background` 要素を取得できなくなる可能性がある気がしています。

Reply:
複数個 Dialog が存在したら `Selector('[data-test=dialog-content]')` で取得できなくなりそうって思ってエイヤしちゃいました。ﾃﾍﾍﾟﾛ。

Reply:
あ、複数個あっても `Selector('[data-test=dialog-content]')` で取得できるコンテンツは `data-test="dialog-content"` を指定している要素だけなので、元の記述は「そのコンテンツを囲っている `background`」を取得する意図で書かれていますね。

Reply:
あーたしかに DialogWrapper.test.ts では content の識別子がそれぞれ違うもの（ data-test="message-dialog-content" etc... ）ですね。ここを勘違いしてました 🙏 

コンテキスト維持するように修正します！

Reply:
@wmoai E2E、id をコンテキストにする形に修正しましたー。

Reply:
ちなみにこちらがクラス名のみの参照となっているのは、 Story 上存在するダイアログは一つだけなのでクラス名参照で問題ないという判断でしょうか？

現状は特にこのままでも動作上問題ないとは思いますが、 Story 内のダイアログの数に依存させたくないと考えるのであれば、先に指摘させていただいた方法と同様に

```suggestion
  const background = content
    .parent('.smarthr-ui-Dialog-wrapper')
    .find('.smarthr-ui-Dialog-background')
```

と記述することで特定の backgorund 要素を参照できると思います。

Reply:
@wmoai 特定のbackground 要素を参照する形にしました！